app/models/concerns/effective_devise_user.rb

```diff
@@ -46,13 +46,13 @@ def effective_devise_user
     end
 
     # Devise invitable ignores model validations, so we manually check for duplicate email addresses.
-    before_save(if: -> { new_record? && invitation_sent_at.present? }) do
+    before_save(if: -> { new_record? && try(:invitation_sent_at).present? }) do
       if email.blank?
         self.errors.add(:email, "can't be blank")
         raise("email can't be blank")
       end
 
-      if self.class.where(email: email.downcase.strip).exists?
+      if !respond_to?(:alternate_email) && self.class.where(email: email.downcase.strip).exists?
```

Comment:
I don't understand why we'd want to skip the email validation here. why?

Reply:
I've added it before adding a  check to `self.errors.added?` - Jus tested and it's safe to remove, thanks!